apps/ledger-live-mobile/src/screens/Onboarding/steps/postWelcomeSelection.tsx

```diff
@@ -29,7 +29,7 @@ function PostWelcomeSelection() {
   const openModal = () => {
     setModalOpen(true);
     track("button_clicked", {
-      button: "I don’t have a Ledger yet",
+      button: "I don't have a Ledger yet",
```

Comment:
Not sure if we need to ping someone as it might break some dashboards

Reply:
Interesting, I will change it back though. I thought it was a wrong input.